services/service-core/src/auth/keycloak.service.ts

```diff
@@ -43,4 +77,21 @@ export class KeycloakService {
       username: decodedToken['preferred_username']
     };
   }
+
+  async editUser(id: string, userInput: object) {
+    const AccessToken = await this.getAdminToken();
+
+    const updateUser = await fetch(`${this.adminUrl}/users/${id}`, {
+      method: 'PUT',
+      headers: {
+        'Content-Type': 'application/json',
+        Authorization: `Bearer ${AccessToken}`
+      },
+      body: JSON.stringify(userInput)
```

Comment:
userInput schema implies that all fields are optional. what if the request is sent without any fields completed? should it be still sent to keycloak?

in general, is validation sufficient here? what if there's any malformed or nonsensical input? should we validate it before sending to keycloak?